docs/technical-setup/environment-setup/install-r-rstudio.md

```diff
@@ -42,26 +42,106 @@ These tools are available from the [R for macOS website](https://mac.r-project.o
+
+1. Finally, copy and paste this line into the Ubuntu terminal.
+    ```sh
+    echo "\noptions(repos = list(CRAN='https://p3m.dev/cran/__linux__/jammy/latest'))" >> ~/.Rprofile
+    ```
+      - This line sets the default R package repository to P3M, the Posit Public Package Manager, instead of the default of CRAN in your [`.Rprofile`](https://support.posit.co/hc/en-us/articles/360047157094-Managing-R-with-Rprofile-Renviron-Rprofile-site-Renviron-site-rsession-conf-and-repos-conf) file.
+       - It will _dramatically_ streamline R package installation by providing you with pre-built package binaries, removing the need to install lots of additional system library dependencies on your computer.
 
-2. Next, you need to install Rtools, which provides additional tools that will let you build R packages that you are likely to encounter in OpenScPCA.
-    - Navigate to the [Rtools download page](https://cran.r-project.org/bin/windows/Rtools/)
-    - Click the Rtools version that matches the R version you just downloaded
-    - On the next page, click the link `RtoolsXY installer` (where `XY` is the specific version you clicked) to download Rtools, and follow all installation instructions
--->
 
 
 ## Install the RStudio IDE
 
+### Install the RStudio IDE on macOS
+
 If you do not already have RStudio installed, you need to [install the free RStudio Desktop](https://posit.co/download/rstudio-desktop/).
 This website shows Step 1 as installing R itself, which you can skip and proceed straight to Step 2, Install RStudio.
 
 Click the download link and follow all instructions to complete the installation.
 
+
+### Install RStudio Server on Windows with WSL2
+
+Because there is no native way to use the RStudio IDE within WSL2's Ubuntu operating system, you will instead need to install the RStudio Server.
+
+This will provide essentially the same experience as working with RStudio Desktop, except you will access the IDE through your browser instead of through a built-in GUI.
+
+To install RStudio Server, run these commands in the Ubuntu terminal:
+
+1. Download the RStudio Server installer file:
+
+    ```sh
+    wget https://rstudio.org/download/latest/stable/server/jammy/rstudio-server-latest-amd64.deb
+    ```
+
+1. Install RStudio Server:
+
+    ```
+    sudo apt install ./rstudio-server-latest-amd64.deb
+    ```
+
+    Note that this line will also start the server automatically when the installation is complete.
+
+1. You can now safely remove the installer file:
+
+
+    ```sh
+    rm rstudio-server-latest-amd64.deb
+    ```
+
+
+#### Using the RStudio Server
+
+To access the RStudio Server, the server must be running.
+You can start and stop the server by running the following lines in Ubuntu terminal.
+We recommend that you stop the server after each coding session:
```

Comment:
Do we? I'm not sure there is much cost to leaving it open.

Reply:
Yeah, probably not.